### PR TITLE
add userList

### PR DESCRIPTION
**Pull request by ShivanshSinghnnovative**

### CODE REVIEW

File: src/composables/userListandCreateUser.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    };
+
+    return {
+        usersListData,
+        getNextUsers,
+        getPreviousUsers,
+        isLoading,
+        limits,
+        stopPageIncrement: computed(() => {
```
Comment (PiyushInnotech) [inline src/composables/userListandCreateUser.js]: why write 2 times same thing

File: src/pages/listUser.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        {{ users.firstName }} {{ users.lastName }}
+      </h1>
+      <h1 class="p-3">
+        {{ users.email }}
+      </h1>
+      <h1 class="p-3 mr-4">
+        {{ getRole(users.roleId) }}
+      </h1>
+    </div>
+    <div class="flex mt-4 absolute right-6">
```
Comment (PiyushInnotech) [inline src/pages/listUser.vue]: make a separate component for pagination

File: src/store/index.js [hunk 3]
```diff
@@ -12,11 +13,15 @@ export default createStore({
     singlePoll: null,
     showMore: true,
     pageNo: ref(1),
+    userPageNo: ref(1),
   },
   mutations: {
     setRoles(state, roles) {
       state.roles = roles;
     },
+    setUser(state, userList) {
```
Comment (PiyushInnotech) [inline src/store/index.js]: change function name

File: src/components/paginationDiv.vue [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            {{ option.label }}
+        </option>
+    </select>
+</template>
+
+<script setup>
+import { usersList } from "../composables/userListandCreateUser.js";
+import { defineEmits, defineProps } from "vue";
+defineProps(["userPageNo"])
+defineEmits([" getNextUsers", "getPreviousUsers", "onLimitChange"]);
```
Comment (meanstack9716) [inline src/components/paginationDiv.vue]: remove space @ShivanshSinghnnovative

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

**Pull request merged**